deployment/src/main/java/io/quarkiverse/quinoa/deployment/packagemanager/FrameworkType.java

```diff
@@ -28,6 +28,7 @@ public enum FrameworkType {
     REACT("build", "start", 3000, Set.of("react-scripts", "react-app-rewired", "craco")),
     VUE_LEGACY("dist", "serve", 3000, Set.of("vue-cli-service")),
     VITE("dist", "dev", 5173, Set.of("vite")),
+    SolidJS("dist", "dev", 3001, Set.of("vite")),
```

Comment:
I still don't think this will work because now we have two VITE sets looking in the scripts folder?

Reply:
Let me find a SolidJS example project

Reply:
Ok Solid JS can be use with any buildtool. I get why this is not possible. My bad!